go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -469,3 +566,13 @@ func cleanupSharedGateway() error {
 	cleanupSharedGatewayIPTChains()
 	return nil
 }
+
+func svcToCookie(name string, ip net.IP, port int32) (string, error) {
+	id := fmt.Sprintf("%s%s%d", name, ip.String(), port)
+	h := fnv.New32a()
```

Comment:
openflow cookies are 64-bit so you could use a 64-bit hash here... (just be sure to update the "/32"s to match)

Reply:
oh, I mistakenly thought they were 32-bit. will fix that.